### PR TITLE
366 Progress Bar Navigation - Intro and Confirmation page

### PR DESCRIPTION
**Pull request by nickjg231**
## Description
This PR ensures that the index page and confirmation pages in a form will not show the progress bar component, while the review page and other pages will.

## Original issue(s)
department-of-veterans-affairs/va-forms-system-core#0366

## Screenshots
image
image
image

image

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link to the original github issue has been provided
- [ ] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: examples/burial-form/ReviewPage.jsx [hunk 1]
```diff
@@ -349,7 +350,7 @@ export default function ReviewPage(props) {
   }
 
   return (
-    <>
+    <Page {...props} nextPage="/confirmation">
```
Comment (bjmccotter7192) [inline examples/burial-form/ReviewPage.jsx:353]: Do you need to put a previous page path here?

File: examples/burial-form/ConfirmationPage.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Page} from '@department-of-veterans-affairs/va-forms-system-core';
+
+export default function ConfirmationPage(props) {
+  return (
+    <>
+      <Page {...props}>
```
Comment (bjmccotter7192) [inline examples/burial-form/ConfirmationPage.jsx:7]: Why do we need to use a `Page` component here?

**Review by bjmccotter7192 — COMMENTED** [review]
Couple comments and I think your tests might have been failing.

File: examples/burial-form/ReviewPage.jsx (hunk 1 above)
Comment (nickjg231) [reply examples/burial-form/ReviewPage.jsx:353]: Yep, I should probably add it here

File: examples/burial-form/ConfirmationPage.jsx (hunk 2 above)
Comment (nickjg231) [reply examples/burial-form/ConfirmationPage.jsx:7]: This is simply a placeholder for the ConfirmationPage component until someone actually picks up that card, as I needed it for testing purposes. I will add a note to that task letting them know that the initial scaffolding is there but the UI needs to be finished.

**Review by mlauren — APPROVED** [review]
LGTM 👍

**Pull request merged**